resources/dicts/patrols/general/training.json

```diff
@@ -1155,16 +1155,16 @@
     "chance_of_success": 60,
     "exp": 5,
     "success_text": [
-    "p_l guides app1 to one of their favorite places and gently asks them what is wrong. app1 looks at their feet and whispers that they feel like a failure and that they are not doing enough for the Clan because they are not doing too well at their training. p_l gently tugs them closer to them and tells them that they are not just their failures, failure hurts thats true but we can learn so much from them. All they can do is go out there and make up for them and ensure that they won't make the same mistake twice. They then assure them that they don't think that app1 is a failure and that they love teaching them and that the Clan feels the same. app1 looks at p_l with watery eyes and then tackles them in a hug saying thank you for believing in me.",
-    "p_l gently asks them what is wrong and app1 breaks down curling into a little ball wailing that they are not doing good enough and that they are really trying. p_l immediately pulls them into a hug purring softly. They then tell them a story of their own apprentice days and how they also made a lot of mistakes, app1 looks up disbelieving, p_l chuckles at the looks on their face and promises them that nobody but themselves will remember those mistakes and as long as you learn from them nobody will hold it against them. They nudge the apprentice onto their feet and slowly walks back to camp with them recounting some more embarrassing stories from their kithood causing app1 to be smiling and chuckling by the time they get back to camp."
-    ],
+    "p_l guides app1 to their favorite place and softly asks them what's wrong. app1 whispers that they feel like a failure and that they are not doing good enough. p_l gently tugs them closer to them and tells them that they are not just their failures, failure hurts that's true but we can learn so much from them. All they can do is go out there and ensure that they won't make the same mistake twice. app1 looks at p_l with shiny eyes and cuddles even closer.", 
```

Comment:
Could adjust to:
"p_l guides app1 to their favorite place and softly asks them what's wrong. app1 whispers that they feel like a failure and that they are not doing good enough. p_l gently tugs them **closer and** tells them that they are not just their failures, failure hurts, that's true, but we can learn so much from them. All they can do is go out there and ensure that they won't make the same mistake twice. app1 looks at p_l with shiny eyes and cuddles even closer."
to make some sentences clearer.